bookstore/settings.py

```diff
@@ -90,6 +95,7 @@
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'bookstoredb',
         'USER': 'postgres',
+        'PASSWORD': '123',
```

Comment:
There are two passowrds here. If we merge, everyone will need to remember to delete one and update the pass.